src/SLIP10Node.ts

```diff
@@ -247,11 +251,11 @@ export class SLIP10Node implements SLIP10NodeInterface {
 
   public readonly index: number;
 
-  public readonly chainCodeBuffer: Buffer;
+  public readonly chainCodeBuffer: Uint8Array;
 
-  public readonly privateKeyBuffer?: Buffer;
+  public readonly privateKeyBuffer?: Uint8Array;
 
-  public readonly publicKeyBuffer: Buffer;
+  public readonly publicKeyBuffer: Uint8Array;
```

Comment:
This change will be breaking. Should we rename these variables to `*Bytes` instead of `*Buffer`?

Reply:
We're gonna do this in a follow-up.